src/test/java/com/playtika/automation/school/test/framework/ServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package com.playtika.automation.school.test.framework;
+
+import com.playtika.automation.school.test.framework.action.AuthActions;
+import com.playtika.automation.school.test.framework.action.NotesActions;
+import com.playtika.automation.school.test.framework.action.RegistrationActions;
+import com.playtika.automation.school.test.framework.configuration.AuthConfiguration;
+import com.playtika.automation.school.test.framework.configuration.NotesConfiguration;
+import com.playtika.automation.school.test.framework.configuration.RegistrationConfiguration;
+import com.playtika.automation.school.test.framework.pojo.responses.AuthResponse;
+import feign.FeignException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.*;
+
+@EnableAutoConfiguration
+@SpringBootTest(classes = {RegistrationConfiguration.class, AuthConfiguration.class, NotesConfiguration.class})
+public class ServiceTest {
+
+    @Autowired
+    private RegistrationActions registrationActions;
+
+    @Autowired
+    private AuthActions authActions;
+
+    @Autowired
+    private NotesActions notesActions;
+
+    private final String updatedContent = "Updated Content";
+    private Integer secondNoteId;
+    private String mail;
+    private String password;
+    private AuthResponse authResponse;
+
+    @BeforeEach
+    void setUp() {
+        mail = registrationActions.generateRandomMail();
+        password = registrationActions.generateRandomPassword();
+        registrationActions.getRegistrationData(mail, password);
+        authResponse = authActions.authUser(mail, password);
+    }
+
+    @Test
+    void serviceTest() {
```

Comment:
bad naming for test method. All methods should start with verb and contain goal of method